ern-api-impl-gen/src/generators/android/ApiImplMavenGenerator.js

```diff
@@ -124,6 +142,7 @@ export default class ApiImplMavenGenerator implements ApiImplGeneratable {
         }
         log.debug(`Api implementation files successfully generated for ${api.apiName}Api`)
       }
+      return {editableFiles}
```

Comment:
Is there a point in returning the result in an object considering it is a single return value ? 

Doing

```javascript
return editableFiles
```

and at the calling site :

```javascript
const editableFiles = await this.generateRequestHandlerClasses(apiDependency, paths, apis)
```

makes it a bit less clumpsy